Sources/Honeycomb/Session/HoneycombSessionIdLogRecordProcessor.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+
+struct HoneycombSessionIdLogRecordProcessor: LogRecordProcessor {
+    private var sessionManager: HoneycombSessionManager
+    private var nextProcessor: LogRecordProcessor
+
+    init(nextProcessor: LogRecordProcessor, sessionManager: HoneycombSessionManager) {
+        self.nextProcessor = nextProcessor
+        self.sessionManager = sessionManager
+    }
+
+    public func onEmit(logRecord: ReadableLogRecord) {
+        var enhancedRecord = logRecord
+
+        enhancedRecord.setAttribute(
+            key: "session.id",
```

Comment:
Does swift have a semantic constant we can rely on?

Reply:
I'll see if we can update that in the span processor as well.